web_widget_x2many_2d_matrix/readme/HISTORY.rst

```diff
@@ -0,0 +1,5 @@
+12.0.2.0.0 (2018-12-07)
```

Comment:
would be nice to catch the commit date (or merge date) of the v12 migration to add the 1st version before